package.json

```diff
@@ -36,9 +36,11 @@
     "binary-search-tree": "0.2.6",
     "bs-ant-design-alt": "2.0.0-alpha.33",
     "bs-css": "11.0.0",
+    "bs-fetch": "^0.5.2",
     "bs-moment": "0.4.4",
-    "bs-platform": "7.2.2",
+    "bs-platform": "^7.2.2",
```

Comment:
why did you add this? The bucklescript version can be a bit fickle.

Reply:
I don't remember; doesn't seem to be necessary. Will keep bucklescript's fickleness in mind.